### DISCUSSION TITLE
Work required to complete project

### DISCUSSION THREAD

**Discussion by captaincoordinates** [sparkgeo/SAR-BC-feature-extract#8] [General]
Add thoughts as comments so that we're not all editing the description at the same time

**Comment by captaincoordinates** [comment]
Investigate performance issues querying FGB data from S3. Suspect that GDAL might be copying the whole dataset locally before querying.

**Comment by captaincoordinates** [comment]
From @cuttlefish: currently a manually-created admin user in the AWS account to deploy, should be using account in the parent org but problems here. Currently only private Sparkgeo repos have access to bot creds and this is difficult to address. No plans to fix right now.

**Comment by brianbancroft** [comment]
One thing that would help the UI is that if you changed the `list` endpoint to return an array of objects:

```
[
  {
      name: 'trails', (string, exactly as expected to receive in backend. lowercase preferred for styling purposes0
      type: 'line' (one of 'line', 'polygon', 'point')
  }
]
```

**Comment by captaincoordinates** [comment]
Open Telemetry change currently in a local branch, should be pushed soon. Some local issues encountered during development. @cuttlefish may continue to investigate.

**Comment by captaincoordinates** [comment]
UI deployment not yet addressed, build / deploy should be relatively simple

**Comment by captaincoordinates** [comment]
Unauthenticated fgb proxy endpoint would need changes to how the fgb data is queried before header requirement added
